src/main/java/mekanism/common/entity/AllayRobitSkin.java

```diff
@@ -0,0 +1,28 @@
+package mekanism.common.entity;
+
+import mekanism.api.robit.RobitSkin;
+import mekanism.common.Mekanism;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class AllayRobitSkin extends RobitSkin {
+
+    public AllayRobitSkin() {
+        super(new ResourceLocation[]{
+                Mekanism.rl( "allay"), Mekanism.rl("allay2")}
+        );
+    }
+
+    @Nullable
+    @Override
+    public ResourceLocation getCustomModel() {
+        return Mekanism.rl("item/robit_allay");
+    }
+
+    @Override
+    public boolean isUnlocked(@NotNull Player player) {
+        return false;
+    }
```

Comment:
Unneeded this is just what super is, unless we want to try require finding an allay before players can use this

Reply:
that would be true, false is not unlocked (so only available by name)

Reply:
Ah hmm that is true I need to think on this more, given setting the name to allay any only making it change the skin if it is on base feels weird. Part of me would rather there is actually some unlock condition  and setting by name doesn't adjust the skin

Reply:
Entitiy Kills are tracked, so killing an allay should be a possible unlock condition.
writing that didn't feel great tbh

Reply:
Part of me initially was thinking player has advancement `husbandry/allay_deliver_item_to_player` but to check that needs a server player and I don't think we are guaranteed that in context everywhere we call it (though if we were then we could easily changed during the breaking change window to having the param be a server player... but I don't believe we can) Though I wonder if the context is accurate enough to if server player check and otherwise assume true on the client side? That might work honestly as I don't think it tends to be called with the player on the client side.

Reply:
I couldn't find clientside validation

Reply:
Wait there is

Reply:
Should a chat message appear on completion of the advancement, that this skin is now available?

Reply:
Na, I don't think that is needed. If you really want you could put a `//TODO: Do we eventually want to make a system for announcing unlocks, maybe using toast notifications` but I don't think there is any real need to *do* so  currently